awscli/formatter.py

```diff
@@ -91,10 +92,11 @@ def _format_response(self, command_name, response, stream):
         # the response will be an empty string.  We don't want to print
         # that out to the user but other "falsey" values like an empty
         # dictionary should be printed.
-        if response:
```

Comment:
Would it be possible to just do `if response or response == 0:`, or are we worried about other types that are falsey but should show up in the output?